test/test.js

```diff
@@ -567,6 +572,32 @@ async function run () {
     t.equal(location, '/api/something')
   })
 
+  test('location headers is preserved when disableInternalRewriteLocationHeader option is defined', async t => {
+    const proxyServer = Fastify()
+
+    proxyServer.register(proxy, {
+      upstream: `http://localhost:${origin.server.address().port}`,
+      prefix: '/my-prefix',
+      disableInternalRewriteLocationHeader: true
```

Comment:
Could you add a dedicated test for the `false` case?

Reply:
Sure, I can add a dedicated test.  

However, there is already a test for this case.  
See `test('rewrite location headers')` from : https://github.com/fastify/fastify-http-proxy/blob/master/test/test.js#L544
It implicitly tests the `false` case since this is the default value of the new option.

Do you want an explicit test?

Reply:
Nope,
thks for pointing out